catroid/src/androidTest/java/org/catrobat/catroid/uitest/content/interaction/UserBrickTest.java

```diff
@@ -545,14 +545,14 @@ public void testBackPackAndUnPackFromDifferentProgrammes() {
 		backPackFirstUserBrickWithContextMenu(DEFAULT_USERBRICK_GROUP_NAME);
 		assertTrue("Userbrick wasn't backpacked!", solo.waitForText(DEFAULT_USERBRICK_GROUP_NAME, 0, TIME_TO_WAIT_BACKPACK));
 
-		UiTestUtils.switchToProgrammBackground(solo, UiTestUtils.PROJECTNAME1, "cat");
+		UiTestUtils.switchToProgrammeBackground(solo, UiTestUtils.PROJECTNAME1, "cat");
```

Comment:
What do you mean here? I've renamed it to ..Programme.. which should be the correct wording